test/OpenTelemetry.Instrumentation.Http.Tests/HttpClientTests.Basic.netcore31.cs

```diff
@@ -281,7 +282,8 @@ public async Task HttpClientInstrumentationBacksOffIfAlreadyInstrumented()
                 Method = new HttpMethod("GET"),
             };
 
-            request.Headers.Add("traceparent", "00-0123456789abcdef0123456789abcdef-0123456789abcdef-01");
+            var traceparent = $"00-{traceId}-{parentSpanId}-01";
```

Comment:
nit: please name it either parentId, or spanId. 
https://www.w3.org/TR/trace-context/#parent-id

Reply:
Done in [dfe3520](https://github.com/open-telemetry/opentelemetry-dotnet/pull/3072/commits/dfe352066a7383b4c679ee8bcb72cd34d472b93b)